fuse_optimizers/src/fixed_lag_smoother.cpp

```diff
@@ -218,8 +218,8 @@ void FixedLagSmoother::optimizationLoop()
       auto optimization_complete = ros::Time::now();
       if (optimization_complete > optimization_deadline)
       {
-        ROS_WARN_STREAM("Optimization exceeded the configured duration by " <<
-                        (optimization_complete - optimization_deadline) << "s");
+        ROS_WARN_STREAM_THROTTLE(10.0, "Optimization exceeded the configured duration by "
+                                           << (optimization_complete - optimization_deadline) << "s");
```

Comment:
Spacing looks weird on github

Reply:
Well, that's actually the spacing at get for those two lines with my `clang-format` configuration.

I can change it if you want.

Although it'd be great if we can have a `.clang-format` file in the `fuse` packages. :smiley: